packages/broker/src/ConfigWizard.ts

```diff
@@ -57,7 +63,6 @@ const DefaultConfig: Config = {
         privateKeyFileName: null,
```

Comment:
These `null` values are not needed as null is the default

Reply:
since the `httpServer` property has it's own type the values are required to be present, even if null